src/test/java/automationpractice/com/pages/OrderHistoryAndDetailsPage.java

```diff
@@ -0,0 +1,21 @@
+package automationpractice.com.pages;
+
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.support.FindBy;
+
+public class OrderHistoryAndDetailsPage extends PageObject {
+    @FindBy(xpath = "//*[@id='order-list']//*[@class='first_item ']//*[contains(@class,'button-small')]")
+    private WebElementFacade clickDetailsButtonForLastOneOrders;
+
+    @FindBy(xpath = "//*[@id='block-order-detail']//*[contains(text(),'entrance from the yard')]" )
+    private WebElementFacade isCommentAboutAddressInLastOneOrders;
+
+    public void clickDetailsButtonForLastOneOrders() {
+        clickDetailsButtonForLastOneOrders.click();
+    }
+
+    public boolean isCommentAboutAddressInLastOneOrdersVisible() {
+        return isCommentAboutAddressInLastOneOrders.getText().contains("entrance from the yard");
+    }
+}
```

Comment:
Powinieneś stworzyć metodę która Ci zwraca listę komentarzy i w warstwie wyżej sprawdzić czy jest tam "entrance from the yard"

Reply:
Zwrócenie listy wszystkich komentarzy do przesyłki wymaga dla każdego z zamówienia kliknięcia przycisku 'Details' a następnie pobranie komentarza o ile jest. Według mnie jest to zbędne bo chcemy sprawdzić tylko czy w ostatnim jest dodany przez nas komentarz. Proponuje zrobienie tak jak napisałaś, tylko bez tworzenia listy.

Reply:
Możesz sobie zimportować
import static automationpractice.com.data.CommentAboutOrder.*;
i zamiast
CommentAboutOrder.entranceFromTheYard.getComment()
po prostu
entranceFromTheYard.getComment()
bedzie troszke krócej